service/build.gradle

```diff
@@ -16,7 +16,24 @@ plugins {
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 apply plugin: 'info.solidsoft.pitest'
-apply plugin: 'org.owasp.dependencycheck'
+
+subprojects {
```

Comment:
isn't this already in allprojects, why is this needed again?

Reply:
I was playing with the config, I removed that now. Seems to work locally as well.